src/component/archive/ArchiveTable/ArchiveTable.tsx

```diff
@@ -0,0 +1,360 @@
+
+    const [searchParams, setSearchParams] = useSearchParams();
+    const { reviveFeature } = useFeatureArchiveApi();
+
+    const [searchValue, setSearchValue] = useState(
+        searchParams.get('search') || ''
+    );
+
+    const onRevive = useCallback(
+        async (feature: string) => {
+            try {
+                await reviveFeature(feature);
+                await refetch();
+                setToastData({
+                    type: 'success',
+                    title: "And we're back!",
+                    text: 'The feature toggle has been revived.',
+                });
+            } catch (e: unknown) {
+                setToastApiError(formatUnknownError(e));
+            }
+        },
+        [refetch, reviveFeature, setToastApiError, setToastData]
+    );
+
+    const columns = useMemo(
+        () => [
+            {
+                id: 'Seen',
+                Header: 'Seen',
+                width: 85,
+                canSort: true,
+                Cell: FeatureSeenCell,
+                accessor: 'lastSeenAt',
+                align: 'center',
+            },
+            {
+                Header: 'Type',
+                accessor: 'type',
+                width: 85,
+                canSort: true,
+                Cell: FeatureTypeCell,
+                align: 'center',
+            },
+            {
+                Header: 'Feature toggle name',
+                accessor: 'name',
+                searchable: true,
+                minWidth: 100,
+                Cell: ({ value, row: { original } }: any) => (
+                    <HighlightCell
+                        value={value}
+                        subtitle={original.description}
+                    />
+                ),
+                sortType: 'alphanumeric',
+            },
+            {
+                Header: 'Created',
+                accessor: 'createdAt',
+                width: 150,
+                Cell: DateCell,
+                sortType: 'date',
+            },
+            {
+                Header: 'Archived',
+                accessor: 'archivedAt',
+                width: 150,
+                Cell: FeatureArchivedCell,
+                sortType: 'date',
+            },
+            ...(!projectId
+                ? [
+                      {
+                          Header: 'Project ID',
+                          accessor: 'project',
+                          sortType: 'alphanumeric',
+                          filterName: 'project',
+                          searchable: true,
+                          maxWidth: 170,
+                          Cell: ({ value }: any) => (
+                              <LinkCell
+                                  title={value}
+                                  to={`/projects/${value}`}
+                              />
+                          ),
+                      },
+                  ]
+                : []),
+            {
+                Header: 'Status',
```

Comment:
I'm pretty sure this should be `State` instead 🤔

Reply:
I have Status in the design, and the design was based on how was implemented, but i need to doublecheck what makes more sense :) Status or State